### DISCUSSION TITLE
Could  we get a "show populated without my instrument"-filter?

### DISCUSSION THREAD

**Discussion by andy1239** [jamulussoftware/jamulus#2084] [Ideas]
SSIA ;-)

using the "#" filters excludes empty  servers which is great! 

I'd really like an aditional function "show populated servers without <text>..." such that you can find a server missing your instrument.

Kind regards
Andy

**Comment by pcar75** [comment]
In case you don't know , https://explorer.jamulus.io/  shows participants with their instrument tag if any.

However, in session, I often find "Listener"-tagged participants who are singers and others (or the same ones) who are also playing one or more instruments.
Of course, guitarists are plentyful  :) , and drummers / percussionnists and bassist are a bit scarce (might have to do with greater sensitivity to latency ... for a rhythym section ? ).

Just a few thoughts.
So ... is it worthwhile ?!

**Reply by pljones** [reply]
I guess it depends where and when you're playing.  Most evenings across Europe it's hard to find a jam that's not already got a drummer... so it seems to me as a drummer :).

But yes, if someone else has joined a server and said "I want to drum", having a filter to show non-empty sessions without a drummer would be handy.  `~`
